packages/js/src/utils/generators.ts

```diff
@@ -0,0 +1,36 @@
+/** @link [Proof Key for Code Exchange by OAuth Public Clients](https://datatracker.ietf.org/doc/html/rfc7636) */
+
+import { fromUint8Array } from 'js-base64';
+
+const DEFAULT_RANDOM_DATA_LENGTH = 64;
+
+/**
+ * @param length The length of the raw random data.
+ */
+function generateRandomString(length: number) {
```

Comment:
```suggestion
const generateRandomString = (length = DEFAULT_RANDOM_DATA_LENGTH) =>
```

Reply:
also we do not have the convention to use UPPER_CASE for constants in js. camelCase would be fine.

Reply:
Fixed, PTAL.